src/branding-bar/BrandingBar.js

```diff
@@ -1,8 +1,62 @@
 import React from "react";
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 
-const BrandingBarContainer = styled.header``;
+import LogoIconSrc from "../assets/icons/recidiviz_logo.svg";
+import { LinkPill } from "../pill";
+
+const brandingBarFlexProperties = css`
+  align-items: center;
+  display: flex;
+`;
+
+const BrandingBarContainer = styled.header`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 50px;
+`;
+
+const BrandingBarHeader = styled.div`
+  ${brandingBarFlexProperties}
+`;
+
+const Logo = styled.img``;
+
+const BrandingBarTitle = styled.h1`
+  color: ${(props) => props.theme.colors.heading};
+  font: ${(props) => props.theme.fonts.display};
+  display: inline;
+  margin-left: 16px;
+`;
+
+const BrandingBarLinkContainer = styled.div``;
+
+const BrandingBarLinksList = styled.ul`
+  ${brandingBarFlexProperties}
+  justify-content: space-between;
+`;
+
+const BrandingBarLink = styled.li`
+  list-style-type: none;
+  margin-left: 16px;
+`;
 
 export default function BrandingBar() {
-  return <BrandingBarContainer>Branding bar</BrandingBarContainer>;
+  return (
+    <BrandingBarContainer>
+      <BrandingBarHeader>
+        <Logo alt="Recidiviz" src={LogoIconSrc} />
+        <BrandingBarTitle>North Dakota Corrections</BrandingBarTitle>
+      </BrandingBarHeader>
+      <BrandingBarLinkContainer>
+        <BrandingBarLinksList>
+          <BrandingBarLink>
+            <LinkPill href="#">Share</LinkPill>
```

Comment:
These pill-based links look fine on the Desktop, but look pretty janky on mobile due to how much space they take up.  Do we have any thoughts on how we'd like to approach mobile treatments for this app?  Presumably using media queries, but more specifically do we know if styled-components natively jives well with mobile development?

Reply:
We need mobile design to be solid at launch since overall polish at launch is going to be important. We should ask Humphrey if there are mobile-first designs anywhere, or if he and Juan have specific adaptations for any elements that may not be mobile-friendly.

As for your last question, I can't speak to that myself. @macfarlandian?

Reply:
A proposal:  https://github.com/Recidiviz/public-dashboard/pull/22

Reply:
I gestured vaguely at mobile support when getting the bones of the site in place, so the overall layout should be more or less responsive, though I will admit that I don't believe the map chart I created is all the way there, and it is in general not very polished. (I have focused thus far on trying to make things shrink to fit the screen and not so much on whether they actually look great when they do so – in part because there are no small-screen designs that have been shared out yet. That said, I have yet to see anything that looks truly unworkable on mobile.)

I have been trying to avoid media queries except where they are absolutely necessary, in favor of flexible layouts – e.g. setting minimum sizes for the smallest supported screens and letting things grow and wrap as needed with flexbox rules or percentage rules, as applicable. But for something like this that isn't really structural, I think using media queries is probably appropriate. Styled Components [supports them](https://styled-components.com/docs/faqs#can-i-nest-rules) as part of its nesting syntax, so no problems there.

Reply:
Ok, cool.  Let me re-work the footer away from the [media query proposal](https://github.com/Recidiviz/public-dashboard/pull/22) and instead try to get it working with only flexbox rules